recipes/default.rb

```diff
@@ -0,0 +1,36 @@
+include_recipe 'apt'
+
+type = case node['platform_family']
+       when 'fedora', 'rhel'
+         'rpm'
+       else
+         'deb'
+       end
+
+packagecloud_repo 'StackStorm/stable' do
+  type type
+end
+
+package 'st2mistral'
+
+# ensure directories are managed
+[
+  node['openstack-mistral']['etc_dir']
+].each do |path|
+  directory "creating mistral directory #{path}" do
```

Comment:
I think we can delete this, especially assuming that `node['openstack-mistral']['etc_dir']` is created by packaging itself (hardcoded).